src/redux/actions/index.js

```diff
@@ -34,3 +44,75 @@ export function setLanguageName(language) {
     payload: language,
   };
 }
+
+// learnt screen actions
+
+export function setLearntPhrases(learntPhrases) {
+  return {
+    type: SET_LEARNT_PHRASES,
+    payload: learntPhrases,
+  };
+}
+
+export function addLearntPhrase(phrase) {
+  return async dispatch => {
+    const storedLearntPhrases = await getStoreItem(LEARNT_PHRASES_KEY);
+    let dataToStore = null;
+    if (!storedLearntPhrases) {
+      dataToStore = [phrase];
+    } else {
+      dataToStore = [...storedLearntPhrases, phrase];
+    }
+    await setStoreItem(LEARNT_PHRASES_KEY, dataToStore);
+    dispatch(setLearntPhrases(dataToStore));
+    return Promise.resolve();
+  };
+}
```

Comment:
Why this is here ?
`addLearntPhrase` is part of the other task, this branch should have changes only related to 'adding new phrases'